src/main/java/org/spongepowered/plugin/meta/version/package-info.java

```diff
@@ -22,22 +22,5 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package org.spongepowered.plugin.meta.util;
-
-import java.lang.annotation.ElementType;
-import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
-
-import javax.annotation.Nonnull;
-import javax.annotation.meta.TypeQualifierDefault;
-
-@Nonnull
-@TypeQualifierDefault({
-        ElementType.FIELD,
-        ElementType.METHOD,
-        ElementType.PARAMETER
-})
-@Retention(RetentionPolicy.RUNTIME)
-public @interface NonnullByDefault {
-
-}
+@org.checkerframework.framework.qual.DefaultQualifier(org.checkerframework.checker.nullness.qual.NonNull.class)
+package org.spongepowered.plugin.meta.version;
```

Comment:
This specific package shouldn't be `@NonNull` by default. The reason is that it is directly sourced from the Maven source code with only little modifications. As such it is not appropriately annotated with `@Nullable` where necessary.

Reply:
Done.